src/app.js

```diff
@@ -1,11 +1,14 @@
+
 import express from 'express';
-import auth from './routes/auth.route';
+import morgan from 'morgan';
+import authRoute from './routes/auth/auth.route';
```

Comment:
@victor-abz  I don't think the auth folder is necessary anymore since we removed that JSDocs.

Reply:
ok